etc/picongpu/aris-grnet/gpu_picongpu.profile.example

```diff
@@ -32,7 +32,6 @@ module load picongpu
 #module load hdf5/1.8.17/gnu
 # Other Software ##############################################################
 #
-# module load zlib/1.2.8
```

Comment:
could you please keep in all profiles `zlib` on most systems the module system is not correctly loading the dependencies of HDF5, therefore we should keep loading zlib in all profiles.